src/helpers/contentWrapper.js

```diff
@@ -0,0 +1,67 @@
+import { sortBy } from 'lodash'
+import { useQuery } from 'cozy-client'
+import {
+  allContacts,
+  contactsbyFamilyNameGivenNameEmailCozyUrl,
+  contactsWithoutIndexes
+} from '../helpers/queries'
+import { updateIndexFullNameAndDisplayName } from '../helpers/contacts'
+
+/**
+ * Fetches only contacts with indexes.byFamilyNameGivenNameEmailCozyUrl,
+ * and fetches only contacts without indexes.byFamilyNameGivenNameEmailCozyUrl
+ * and concat results
+ * @returns {object} query result with data
+ */
+export const fetchContactsWithAndWithoutIndexes = () => {
+  const resultWithIndexes = useQuery(
+    contactsbyFamilyNameGivenNameEmailCozyUrl.definition,
+    contactsbyFamilyNameGivenNameEmailCozyUrl.options
+  )
+
+  const resultWithoutIndexes = useQuery(
+    contactsWithoutIndexes.definition,
+    contactsWithoutIndexes.options
+  )
+
+  const fetchStatus =
+    resultWithIndexes.fetchStatus === 'loaded' &&
+    resultWithoutIndexes.fetchStatus === 'loaded'
+      ? 'loaded'
+      : 'loading'
+
+  const hasMore = resultWithIndexes.hasMore || resultWithoutIndexes.hasMore
+
+  const fetchMore = resultWithIndexes.hasMore
+    ? resultWithIndexes.fetchMore
+    : resultWithoutIndexes.fetchMore
```

Comment:
What happens if both have fetchMore?

Reply:
Both should have fetchMore anyway, at least it will be a function doing nothing.
As long as `hasMore` isn't at `false`, it must fetch anyway. If it's the first request that has `hasMore`, it's its `fetchMore` that's passed, otherwise it's the `fetchMore` of the second request.